src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -2115,11 +2115,25 @@ public MuteResult handleMuteRequest(Jid muterJid, Jid toBeMutedJid, boolean doMu
             return MuteResult.ERROR;
         }
 
-        // do not allow unmuting other participants even for the moderator
-        if (!doMute && !muterJid.equals(toBeMutedJid))
+        // process unmuting
+        if (!doMute)
         {
-            logger.warn("Unmute now allowed, mutedJid=" + muterJid + ", toBeMutedJid=" + toBeMutedJid);
-            return MuteResult.NOT_ALLOWED;
+            // when A/V moderation is enabled we need to check the whitelists
+            if (this.chatRoom.isAvModerationEnabled())
+            {
+                if (!this.chatRoom.isMemberAllowedToUnmute(toBeMutedJid, mediaType))
+                {
+                    logger.warn("Unmute not allowed due to av moderation, mutedJid="
+                        + muterJid + ", toBeMutedJid=" + toBeMutedJid);
+                    return MuteResult.NOT_ALLOWED;
+                }
+            }
+            // do not allow unmuting other participants even for the moderator
+            else if (!muterJid.equals(toBeMutedJid))
```

Comment:
It is strange that this behavior is different based on whether moderation is enabled or not.

For the purpose of moderated meetings, clients need to expect that their channel could be unmuted sever-side, thus they must make sure not to send media unless the user initiates it. If they already have that logic, we may as well separate the local mute and the server-side mute, and allow moderators to do the latter. 

But (even if you agree) this is probably outside the scope of this PR.

Reply:
Still, we want to double make sure that clients can not be unmuted remotely without the user's consent in the case of moderation (which includes e.g. someone turning moderation on/off at the right time)

Reply:
We need to talk about it ... I'm confused.

Reply:
It is just this comment that had left from this PR.

Reply:
Ok, it takes me a day to get it, but in the end it came to me :) 
I think I fixed it.